src/ui/components/Library/Team/View/Tests/Overview/TestErrorList.tsx

```diff
@@ -0,0 +1,183 @@
+import { motion } from "framer-motion";
+import orderBy from "lodash/orderBy";
+import { Dispatch, SetStateAction, useState } from "react";
+import { ContextMenuItem, useContextMenu } from "use-context-menu";
+
+import Icon from "replay-next/components/Icon";
+import { TestExecution } from "shared/test-suites/TestRun";
+import LibraryDropdownTrigger from "ui/components/Library/LibraryDropdownTrigger";
+
+import { getTruncatedRelativeDate } from "../../Recordings/RecordingListItem/RecordingListItem";
+import styles from "../../../../Library.module.css";
+
+const MAX_ERRORS_SHOWN = 5;
+const MAX_REPLAYS_SHOWN = 3;
+
+export function TestErrorList({
+  errorFrequency,
+  executions,
+}: {
+  errorFrequency: Record<string, number>;
+  executions: TestExecution[];
+}) {
+  const [selectedError, setSelectedError] = useState<string | null>(null);
+  const [filterByTime, setFilterByTime] = useState<number | null>(null);
+
+  const {
+    contextMenu: contextMenuTimeFilter,
+    onContextMenu: onClickTimeFilter,
+    onKeyDown: onKeyDownTimeFilter,
+  } = useContextMenu(
+    <>
+      <ContextMenuItem disabled onSelect={() => setFilterByTime(1 / 24)}>
+        Last hour
+      </ContextMenuItem>
+      <ContextMenuItem disabled onSelect={() => setFilterByTime(1)}>
+        Last day
+      </ContextMenuItem>
+      <ContextMenuItem disabled onSelect={() => setFilterByTime(7)}>
+        Last week
+      </ContextMenuItem>
+      <ContextMenuItem disabled onSelect={() => setFilterByTime(30)}>
+        Last month
+      </ContextMenuItem>
+      <ContextMenuItem onSelect={() => setFilterByTime(null)}>All time</ContextMenuItem>
+    </>,
+    { alignTo: "auto-target" }
+  );
+
+  const uniqueErrorCount = Object.entries(errorFrequency).length;
+
+  return (
+    <div>
```

Comment:
This needs overflow. As it is now, if the page is too short to show them all, you can't scroll to see what's cut off. (Test by making your browser page shorter.)
```suggestion
        <div class="overflow-y-auto">
```